mirage/config.js

```diff
@@ -105,6 +105,10 @@ export default function (config) {
           parentComment: belongsTo('community-course-stage-solution-comment', { inverse: null }),
         }),
         fakeLogstream: Model.extend({}),
+        institutionMembershipGrant: Model.extend({
```

Comment:
Are you sure we need this? Can't these relationships not be inferred? The other ones in this file use relationships that cannot be inferred for some reason (I forget exactly why but I think currentUserDownvotes / currentUserUpvotes is the reason)